whisperv6/config.go

```diff
@@ -21,6 +21,7 @@ type Config struct {
 	MaxMessageSize                        uint32  `toml:",omitempty"`
 	MinimumAcceptedPOW                    float64 `toml:",omitempty"`
 	RestrictConnectionBetweenLightClients bool    `toml:",omitempty"`
+	NoConfirmations                       bool    `toml:",omitempty"`
```

Comment:
I am not a fan of this inverted implicitly enabled configuration fields :P

Reply:
this is here only for testing. i don't see any use case to disable confirmations if version of the client supports them. thats why it is inverted